tests/pos/i12739.scala

```diff
@@ -0,0 +1,15 @@
+object O {
+
+  class CA[A](var x: A)
+  type C = CA[_]
+
+  val c: CA[_] = CA(5)
```

Comment:
Is there a reason this is `CA[_]` rather than `C`, like in the report?  Obviously you'd need to make the RHS work with `C` (the report used `???`).

Reply:
It's an accidental change, now updated.